service/worker.go

```diff
@@ -427,8 +427,14 @@ func (s *WorkerService) initiateTxSignWithVerifier(ctx context.Context, signRequ
 		return err
 	}
 
+	verifierURL := s.cfg.Server.VerifierURL
+	if verifierURL == "" {
+		// Only append port for local development
+		verifierURL = fmt.Sprintf("http://localhost:%d", s.verifierPort)
+	}
```

Comment:
I don't think we need this - for local dev, just set in the config file?

Reply:
ok. removing